vispy/app/backends/tests/test_ipynb_util.py

```diff
@@ -3,10 +3,10 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 import numpy as np
 
-from vispy.app.backends._ipynb_util import (_extract_buffers,
+from ....app.backends._ipynb_util import (_extract_buffers,
```

Comment:
the test files are actually thought of more as scripts (note especially the `run_tests_if_main()` at the bottom that actually allows them to run if you call them) so all test script imports should be absolute